src/NServiceBus.AcceptanceTests/Performance/TimeToBeReceived/When_TimeToBeReceived_has_expired_convention.cs

```diff
@@ -23,18 +24,45 @@ public class Context : ScenarioContext
             public bool WasCalled { get; set; }
         }
 
+        class DelayReceiverFromStarting : Feature
+        {
+            protected override void Setup(FeatureConfigurationContext context)
+            {
+                context.Container.ConfigureComponent<DelayReceiverFromStartingTask>(DependencyLifecycle.SingleInstance);
+                context.RegisterStartupTask(b => b.Build<DelayReceiverFromStartingTask>());
+            }
+        }
+
+        class DelayReceiverFromStartingTask : FeatureStartupTask
+        {
+            protected override async Task OnStart(IMessageSession session)
+            {
+                await session.SendLocal(new MyMessage());
+                await Task.Delay(TimeSpan.FromSeconds(5));
+            }
+
+            protected override Task OnStop(IMessageSession session)
+            {
+                return Task.FromResult(0);
+            }
+        }
+
         public class Endpoint : EndpointConfigurationBuilder
         {
             public Endpoint()
             {
-                EndpointSetup<DefaultServer>(c=>c.Conventions().DefiningTimeToBeReceivedAs(messageType =>
+                EndpointSetup<DefaultServer>(c =>
                 {
-                    if (messageType == typeof(MyMessage))
+                    c.EnableFeature<DelayReceiverFromStarting>();
```

Comment:
Side note: Features can be made "enabled by default" but I like us being explicit when we use them for testing like this :+1: